cmd/ci-operator-prowgen/main.go

```diff
@@ -277,6 +305,78 @@ func generatePodSpecTemplate(info *config.Info, release string, test *cioperator
 	return podSpec
 }
 
+func generatePodSpecRandom(info *config.Info, test *cioperatorapi.TestStepConfiguration) *kubeapi.PodSpec {
+	cmd := []string{"-c", fmt.Sprintf(openshiftInstallerRandomCmd, test.As)}
+	podSpec := generatePodSpec(info, test.As, []string{"bash"}, cmd)
+	profiles := []string{"aws", "azure4", "vsphere"}
```

Comment:
I think this should be extracted somewhere more prominent to be more easily found when platforms are added / removed.

Reply:
Hmm, I certainly should have used the constants we already have, now that I think about it. I'll move this to a constant.